packages/core/src/Accordion/Accordion.tsx

```diff
@@ -30,6 +29,7 @@ export const Accordion = ({
 }: IAccordion) => {
   return items ? (
     <RadixAccordion.Root
+      // @ts-ignore
```

Comment:
How Radix sets up their TS type makes it impossible (at least to my knowledge) to satisfy the type check... not sure why this is possible with TS or at least so difficult and unintuitive to resolve